### PR TITLE
MD EITC

### LINKED ISSUE #942: Add Maryland EITC
Matches the federal EITC at 50% (nonrefundable) or 45% (refundable). Also removes the federal minimum age requirement.

https://www.marylandtaxes.gov/statepayroll/Static_Files/2021_memos/2021-Earned-Income-Credit-EIC.pdf

They also have an EITC Assistant calculator at https://interactive2.marylandtaxes.gov/EITCAssistant/

@ngpsu22

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Fixes #942 

@MaxGhenis @fedderw - in progress, but I found a way to have the federal EITC (modified to remove the age condition for childless individuals) inherit from the baseline EITC.

### CODE REVIEW

**Comment by nikhilwoodruff** [comment]
This is ready for review. I've left out the local deduction because I think we need the local tax rates in first. But this handles the non-refundable and refundable State income tax credits.

File: openfisca_us/variables/gov/states/md/tax/income/credits/eitc/md_state_refundable_eitc.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+class md_state_refundable_eitc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MD EITC refundable State tax credit"
+    unit = USD
+    documentation = "Refundable EITC credit reducing MD State income tax."
+    definition_period = YEAR
+    reference = "https://casetext.com/statute/code-of-maryland/article-tax-general/title-10-income-tax/subtitle-7-income-tax-credits/section-10-704-effective-until-6302023-for-earned-income"
```
Comment (MaxGhenis) [inline openfisca_us/variables/gov/states/md/tax/income/credits/eitc/md_state_refundable_eitc.py:11]: Have you seen the sequence of the nonrefundability? The CDCC is also partly nonrefundable. We might have to go to the forms.

**Review by MaxGhenis — APPROVED** [review]

File: openfisca_us/parameters/gov/states/md/tax/income/credits/eitc/refundable_match.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+description: Percentage of the excess of the non-refundable MA State EITC over State income tax that is refundable.
```
Comment (MaxGhenis) [inline openfisca_us/parameters/gov/states/md/tax/income/credits/eitc/refundable_match.yaml:1]:
```suggestion
description: Percentage of the excess of the non-refundable MD State EITC over State income tax that is refundable.
```

File: openfisca_us/variables/gov/states/md/tax/income/credits/eitc/md_state_refundable_eitc.py (hunk 1 above)
Comment (nikhilwoodruff) [reply openfisca_us/variables/gov/states/md/tax/income/credits/eitc/md_state_refundable_eitc.py:11]: Yeah the law has a circular dependency, we'll have to go off forms

Comment (nikhilwoodruff) [reply openfisca_us/variables/gov/states/md/tax/income/credits/eitc/md_state_refundable_eitc.py:11]: [Looks like the EITC is first, so this is fine](https://www.marylandtaxes.gov/forms/21_forms/502.pdf)

**Pull request merged**